pkg/webui/console/components/location-form/index.js

```diff
@@ -155,10 +175,22 @@ class LocationForm extends Component {
       locationFieldsDisabled,
       allowDelete,
     } = this.props
-    const { error } = this.state
+    const { error, latitude, longitude } = this.state
 
     const entryExists = hasLocationSet(initialValues)
 
+    const marker =
+      latitude && longitude
+        ? [
+            {
+              position: {
+                latitude: Number(latitude),
+                longitude: Number(longitude),
+              },
+            },
+          ]
+        : undefined
+
```

Comment:
I think we should look into avoiding multiline ternaries.
```suggestion
    let marker = undefined
    if (latitude && longitude) {
      marker =  [{
        position: {
          latitude: Number(latitude),
          longitude: Number(longitude),
        },
      }]
```

Reply:
this needs to be fixed in prettier config, otherwise travis will complain

Reply:
The styling is arbitrary, I was talking about replacing multiline ternaries with if statements.

Reply:
aah, sorry noticed the wrong thing